docsrc/content/_template.html

```diff
@@ -4,16 +4,16 @@
     <meta charset="utf-8"/>
```

Comment:
Note that this template does not work well, since It's not the one that @adz has fixed. This is simply the minimal effort conversion of template.cshtml to html.